tests/testsuite/kubevirtresource.go

```diff
@@ -91,8 +130,14 @@ func AdjustKubeVirtResource() {
 			},
 		},
 	}
+	arch := TranslateBuildArch()
+	// Disable CPUManager Featuregate for s390x as it is not supported.
```

Comment:
nit: you can use `DisableFeatureGate` if the arch is s390x

Reply:
How does this interfere with the test suite?